src/containerapp/azext_containerapp/containerapp_sessionpool_decorator.py

```diff
@@ -42,6 +42,7 @@
 class ContainerType(Enum):
     PythonLTS = 0
     CustomContainer = 2
+    NodeLTS = 3
```

Comment:
how do you make container type required?

Reply:
It's not a required field now, I have removed the validation check for container_type.